# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/perl5/use_db3.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Force perl5 to use bdb33, compatibility reasons.

--- ./ext/DB_File/Makefile.PL.orig	2003-09-28 09:29:41.000000000 -0400
+++ ./ext/DB_File/Makefile.PL	2003-09-28 09:31:37.000000000 -0400
@@ -8,15 +8,15 @@
 # OS2 is a special case, so check for it now.
 my $OS2 = "-DOS2" if $Config{'osname'} eq 'os2' ;
 
-my $LIB = "-ldb" ;
+my $LIB = "-ldb-3" ;
 # so is win32
 $LIB = "-llibdb" if $^O eq 'MSWin32' ;
 
 WriteMakefile(
 	NAME 		=> 'DB_File',
-	LIBS 		=> ["-L/usr/local/lib $LIB"],
+	LIBS 		=> ["-L/usr/lib $LIB"],
         MAN3PODS        => {},         # Pods will be built by installman.
-	#INC		=> '-I/usr/local/include',
+	INC		=> '-I/usr/include/db3',
 	VERSION_FROM	=> 'DB_File.pm',
 	OBJECT		=> 'version$(OBJ_EXT) DB_File$(OBJ_EXT)',
 	XSPROTOARG	=> '-noprototypes',
